ocl/kernels/mosaic.cl

```diff
@@ -44,7 +46,9 @@ __kernel void mosaic(__write_only image2d_t img_y_dst,
     barrier(CLK_LOCAL_MEM_FENCE);
     Y.x = sum[l_id_x / blk_size * blk_size];
     for (uint i = 0; i < blk_size; i++) {
-        write_imagef(img_y_dst, (int2)(g_id_x + crop_x, g_id_y * blk_size + i + crop_y), Y);
+        if (g_id_x <= crop_w && g_id_y * blk_size + i <= crop_h) {
```

Comment:
This add judgment in opencl loop,maybe it's not so effcient, Could we filter before the loop?